### ISSUE TITLE
[VSCode Doc] add the requirement for vscode version in readme

### ISSUE THREAD

**Issue by chunyu3** [microsoft/typespec#5425]
The typespec VSCode extension required the VSCode version > 1.94.0. Need to add this perquisite into the Readme.md

https://github.com/microsoft/typespec/blob/fa488db1bfcb9c509815f22bc82020aa0c349014/packages/typespec-vscode/package.json#L29

**Comment by josefree** [comment]
I don't think we should mention that in readme without explanation. If we have any error related, we can set up a troubleshooting by verifying the version of the VS Code.
